### ISSUE TITLE
[Bug] Error when translation file is missing

### ISSUE THREAD

**Issue by gptgit** [KwiatekMiki/passnpingenerator#13]
When the translation file for the user's Windows language is missing, the program prints a message but does not raise an exception, causing the program to continue executing with missing translations.

Steps to Reproduce:

Run the program without the translation file for the user's Windows language.
Observe the program output.
Expected Behavior:
If the translation file for the user's Windows language is missing, the program should raise a FileNotFoundError exception.

Actual Behavior:
The program prints a message but does not raise an exception, resulting in missing translations and potentially incorrect program behavior.

Additional Information:

Operating System: Windows 10
Language: English
Version: 1.0

**Comment by KwiatekMiki** [comment]
Oops, forgot to add `raise`
```
> python main.py
Can't find a translation for your Windows language, using English.
Traceback (most recent call last):
  File "D:\Dane\Dokumenty\GitHub\passnpingenerator\main.py", line 22, in <module>
    with open(f'./lang/{oslang}.json', 'r', encoding='utf-8') as f:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: './lang/en.json
```

**Comment by KwiatekMiki** [comment]
### Fixed in 81fb03a
